onnxruntime/test/perftest/ort_test_session.cc

```diff
@@ -90,8 +90,10 @@ OnnxRuntimeTestSession::OnnxRuntimeTestSession(Ort::Env& env, std::random_device
     session_options.EnableSequentialExecution();
   else
     session_options.DisableSequentialExecution();
-  fprintf(stdout, "Setting thread pool size to %d\n", performance_test_config.run_config.session_thread_pool_size);
-  session_options.SetThreadPoolSize(performance_test_config.run_config.session_thread_pool_size);
+  fprintf(stdout, "Setting intra_op_num_threads to %d\n", performance_test_config.run_config.intra_op_num_threads);
```

Comment:
>fprintf [](start = 2, length = 7)

What's the reason for not using std::cout?

Reply:
No idea. It was previous code :-)

Reply:
1. printf is simpler than cout. You know when it is atomic when it is not.  
2. When you need to deal with locale, std::cout is a mess. Different platform has different behavior, there isn't an easy way to make it always work. 
3. Easier to get translated to the other human languages.

Reply:
If needed, let's resolve this in a separate PR.